privacyidea/static/components/config/views/config.resolvers.ldap.html

```diff
@@ -31,6 +31,24 @@
                    ng-model="params.LDAPURI" required
                    placeholder="ldap://privacyidea.server1, ldap://privacyidea.server2"/>
         </div>
+    </div>
+        <div class="form-group"
+        ng_show="params.LDAPURI.toLowerCase().substring() ===','">
+        <label for="ldap_pooling_strategy"
+            class="col-sm-3 control-label" translate>
+            pooling strategy
+        </label>
+        <div class="col-sm-4">
+            <select class="form-control"
+                    name="ldap_pooling_strategy" id="ldap_pooling_strategy"
+                    ng-model="params.LDAP_POOLING_STRATEGY">
```

Comment:
this needs to be `ng-model="params.SERVERPOOL_STRATEGY"` otherwise it won't be used in the LDAPResolver at all.